examples/simple-request/asyncapi-nats-client/API.md

```diff
@@ -142,34 +118,25 @@ Try to connect to the NATS server which has no authentication
 | host | to connect to |
 | options | to connect with |
 
-<a name="NatsAsyncApiClient+subscribeToStreetlightStreetlightIdCommandTurnon"></a>
-
-### natsAsyncApiClient.subscribeToStreetlightStreetlightIdCommandTurnon(onDataCallback, streetlight_id, flush, options)
-Subscribe to the `streetlight/{streetlight_id}/command/turnon`
+<a name="NatsAsyncApiClient+connectToLocal"></a>
 
-Channel for the turn on command which should turn on the streetlight
+### natsAsyncApiClient.connectToLocal()
+Connects the client to the AsyncAPI server called local.
+Local server used during development and testing
```

Comment:
```suggestion
Connects the client to the AsyncAPI server called local. The local server is used during development and testing.
```

Reply:
This is actually auto-generated from multiple places, i.e. template -> jsdoc -> markdown 😄